youtube_dl/extractor/walla.py

```diff
@@ -57,16 +80,23 @@ def _real_extract(self, url):
                 'url': xpath_text(subtitle, './src'),
             }]
 
+        playlist_url = 'http://walla-s.vidnt.com/walla_vod/_definst_/%s.mp4/playlist.m3u8'
+
         formats = []
+        formats.append(
+            {
+                'url': playlist_url % default_file,
+                'ext': 'mp4',
+                'format_id': 40,
+            }
+        )
+
         for quality in item.findall('./qualities/quality'):
             format_id = xpath_text(quality, './title')
             fmt = {
-                'url': 'rtmp://wafla.walla.co.il/vod',
```

Comment:
I checked this from a german and US IP on contents I know to be blocked, can you please check it or let me know a specific url and geo-location to check?